docs/src/modules/components/Demo.js

```diff
@@ -430,10 +430,10 @@ export default function Demo(props) {
 
   if (demoOptions.bg == null) {
     demoOptions.bg = 'outlined';
-  }
 
-  if (demoOptions.iframe) {
-    demoOptions.bg = true;
+    if (demoOptions.iframe) {
+      demoOptions.bg = true;
+    }
   }
 
```

Comment:
A Docs-infra bug, even if we set a default background for iframe different than for non-iframe demos, we should still be able to customize it.